bin/cylc-trigger

```diff
@@ -109,7 +109,15 @@ def main():
         # Add the 'NN' symlink for latest job file regardless of submit number.
         jobfile_path = os.path.join(job_dir, 'NN', 'job')
 
-        current_time = time.time()
+        # Note: localhost time and file system time may be out of sync,
+        #       so the safe way to detect whether a new file is modified
+        #       or is to detect whether time stamp has changed or not.
+        #       Comparing the localhost time with the file timestamp is unsafe
+        #       and may cause the "while True" loop the follows to hang.
```

Comment:
v. minor issues with this comment: (a) a typo - "*that* follows"; (b) the loop wouldn't *hang*, just abort after `MAX_TRIES` seconds.